### PR TITLE
getL2HashFromL1DepositInfo produces invalid results

### PR DESCRIPTION
**Pull request by AlexBHarley**
I've noticed that occasionally, `getL2HashFromL1DepositInfo` produces invalid results.

Consider [this](https://etherscan.io/tx/0x33faeeee9c6d5e19edcdfc003f329c6652f05502ffbf3218d9093b92589a42c4#eventlog) deposit transaction from Mainnet to OP Mainnet.

op-viem computes the hash to be `0x020ca38d3efb0809f78a79a301cb4b3ff493b3b41b8f537e65734510230ee175` but looking at [Etherscan](https://optimistic.etherscan.io/address/0xF282ED5De6F51854b4f07f5C1DBC8F178ab8a89B#deposittxs) we can see it should be `0xed88afbd3f126180bd5488c2212cd033c51a6f9b1765249bdb738dcac1d0cb41` 

 I've combed through the code to what could be different between op-node and op-viem but I haven't found it yet.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (wilsoncusack) [inline .prettierrc:1]: we use rome :)

Comment (AlexBHarley) [reply .prettierrc:1]: My editor kept making everything messy. Will remove

**Comment by wilsoncusack** [comment]
looking at this now and will revert. Thanks!

**Comment by wilsoncusack** [comment]
fixed in #44 thanks!

**Pull request closed without merging**